tests/mve/gtest_scene.cc

```diff
@@ -0,0 +1,751 @@
+#include <gtest/gtest.h>
+
+#include <cstdio>
+#include <iomanip>
+#include <sstream>
+#include <string>
+
+#include "mve/bundle_io.h"
+#include "mve/scene.h"
+#include "util/exception.h"
+#include "util/file_system.h"
+
+namespace {
+
+/* Cleanup mechanism. Unlinks files on destruction. */
+class OnScopeExit final
+{
+public:
+    ~OnScopeExit();
+    void unlink(std::string str);
+
+private:
+    void unlink_recursive(const std::string& str);
+
+    std::vector<std::string> paths;
+};
+
+/**
+ * Creates a valid scene directory in the system's temporary directory.
+ * This directory has an empty "views" subdirectory and
+ * a bundle file with zero cameras and zero features.
+ * @param view_count Number of views the scene is created with.
+ * @param bundle Bundle to include within the scene or
+ *     nullptr to create a scene without a bundle file.
+ * @return The path to the created scene directory.
+ */
+std::string create_scene_on_disk(const std::size_t view_count,
```

Comment:
In MVE, just use up to 80 chars, and if the line needs to be wrapped, indent with four spaces. Don't align with brackets.
When declaring and defining functions, put a space before the bracket, like `int function (int value)`. If I'd have re-evaluate the style, I'd probably do away with this rule, but at this point let's just be consistent.